share/ci/scripts/linux/yum/install_docs_env.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+
+set -ex
+
+HERE=$(dirname $0)
+
+bash $HERE/install_doxygen.sh latest
+bash $HERE/../install_sphinx.sh latest
+bash $HERE/../install_testresources.sh latest
+bash $HERE/../install_recommonmark.sh latest
+bash $HERE/../install_sphinx-press-theme.sh latest
+bash $HERE/../install_breathe.sh latest
```

Comment:
[Question] In order to avoid 'repeat ourself' anti-pattern (i.e. one day one will be changed but not the other), could you factorize this content with `docs/requirements.txt` ?